lib/beckon.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+
+class Beckon
+  def initialize(start_time, bot, event)
+    @start_time = start_time
+    @spotters = []
+    @beckon_message = bot.send_message(
+      event.channel,
+      "<@&#{ROLE_ID}> a new beckon has appeared!",
+      false,
+      {
+        color: 13_632_027,
+        fields: [
+          {
+            name: "**:stopwatch: Start Time**: #{start_time.strftime("%F %l:%M %p").strip}",
+            value: ""
+          },
+          {
+            name: "<:coolspot:#{COOLSPOT_ID}> | 0",
+            value: ""
+          }
+        ]
+      },
+      nil,
+      {
+        roles: [ROLE_ID]
+      }
+    )
+
+    @beckon_message.react "coolspot:#{COOLSPOT_ID}"
```

Comment:
I don't think we should have side effects in `initialize`. I'd suggest we break this out into another method at some point.

Reply:
addressed yip yip yeeeee